app/design/adminhtml/Magento/backend/Magento_Backend/web/css/source/module/main/_collapsible-blocks.less

```diff
@@ -126,6 +126,11 @@
                     display: block;
                 }
             }
+            &.active {
```

Comment:
Hi!
Currently it does not work as expected. Could you move it few lines up so it would be under same selector as expand-close icon?  
line 118 selector:    
&._show {
        > .fieldset-wrapper-title {
            .admin__collapsible-title
...

Cheers,
Karla